homeassistant/components/arcticspa/manifest.json

```
@@ -0,0 +1,9 @@
+{
+  "domain": "arcticspa",
+  "name": "Arctic Spa",
+  "codeowners": ["@greg-szabo"],
+  "config_flow": true,
+  "documentation": "https://www.home-assistant.io/integrations/arcticspa",
```

```suggestion
  "documentation": "https://www.home-assistant.io/integrations/arcticspa",
  "integration_type": "device",
```
For what I can see, an API key corresponds to exactly one device?

---

I believe that's a correct assumption. I hope eventually the API introduces controls to control multiple devices but right now it looks like one device = one API key.